py_zipkin/transport/firehose_handler.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import socket
+
+import clog
+import staticconf
+
+
+ZIPKIN_CONFIG_NAMESPACE = 'zipkin'
+ZIPKIN_CONFIG_FILE = '/nail/srv/configs/zipkin.yaml'
+ZIPKIN_CONFIG_MIN_RELOAD_INTERVAL = 5
+
+ZIPKIN_FIREHOSE_IP = '169.254.255.254'
```

Comment:
We hardcoded this to make it easier for us, but since we're including it in py_zipkin now, we should make the IP and Port configurable